docs/dev/test-configuration-options.md

```diff
@@ -11,21 +12,25 @@ export const Highlight = ({children, color}) => ( <span style={{
       padding: '0.2rem',
     }}>{children}</span> );
 
-This page includes a list of valid test configuration options for tests run on Sauce Labs.
+See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the code for setting the capabilities to execute a test.
 
-See the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the basic code necessary for executing a test. Those settings as well as all valid parameters are described below.
+Depending on which environment you are running tests in, different options, also known as **Capabilities**, should be set. There are different sets of capabilities for different environments, which can be combined. These configurations are added to the Capabilities or Options classes. Some of these setting are required for a test to run in a given environment, while some are optional. Certain capabilities
+
```

Comment:
"Selenium Capabilities or Options classes (depending on the language)"

Reply:
@walkerlj0 -- could you please italicize in this sentence?

Depending on which environment you are running tests in, different options, also known as _**Capabilities**_ (or options classes),